pkgs/top-level/all-packages.nix

```diff
@@ -9833,9 +9833,19 @@ with pkgs;
     withUtils = false;
   });
 
-  libva = callPackage ../development/libraries/libva { };
-  libva-full = libva.override { minimal = false; };
-  libva-utils = callPackage ../development/libraries/libva-utils { };
+  inherit (callPackage ../development/libraries/libva { })
```

Comment:
There's `callPackages` too for this sort of thing. Better so that `override` works per-package, rather than being lost.